src/cci-preset.ts

```diff
@@ -43,6 +43,14 @@ export const cciPreset = definePreset({
   // We're generating all the recipes
   staticCss: {
     recipes: "*",
+    css: [
```

Comment:
@Khaan25 Why do we have these sizes? Didn't we already fix the icon size issue?

Reply:
Nope, it is not fixing that for some reason :( 

I've hard coded the width and height in main.css file in upstat project

Reply:
Ok, it needs to be figured out